### PR TITLE
Standardised docs

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for the issue #1412

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 55 sec

File: docs/src/user/manual/login-logout/login-page.md [hunk 1]
```diff
@@ -1,34 +1,32 @@
...
+1. **Email**: Enter the users registered email address in this field. This is the email associated with the users account.
 
-2. **Password**: Input your password in this field. Passwords are case-sensitive, so ensure you enter it correctly.
+2. **Password**: Input the users password in this field. Passwords are case-sensitive, so ensure the user enter it correctly.
 
-3. **Remember Me**: Check this box if you want the application to remember your login credentials for future sessions. This feature is optional and can be useful for quick and convenient access.
+3. **Remember Me**: Check this box if the user want the application to remember the users login credentials for future sessions. This feature is optional and can be useful for quick and convenient access.
 
-4. **Login Button**: Click the `LOGIN` button to submit your credentials and access your account.
+4. **Login Button**: Click the `LOGIN` button to submit the users credentials and access the users account.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/login-page.md:15]:
```suggestion
4. **Login Button**: Click the `LOGIN` button to submit the user's credentials and access their account.
```

File: docs/src/user/manual/login-logout/login-page.md [hunk 2]
```diff
@@ -1,34 +1,32 @@
...
 
 5. **Forgot Password**: The `Forgot Password` link opens the [forgot password](forgot-password.md) for reset password.
 ### Remember Me
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/login-page.md:18]:
```suggestion
5. **Forgot Password**: The `Forgot Password` link opens the [forgot password](forgot-password.md) for reset password.

### Remember Me
```

File: docs/src/user/manual/login-logout/login-page.md [hunk 3]
```diff
@@ -1,34 +1,32 @@
...
 
-The "Remember Me" checkbox allows you to opt for the application to remember your login information. When checked, you won't need to re-enter your email and password each time you visit the login page. This is particularly convenient for returning users who want a seamless login experience.
+The `Remember Me` checkbox allows the user to opt for the application to remember the users login information. When checked, the user won't need to re-enter the users email and password each time the user visit the login page. This is particularly convenient for returning users who want a seamless login experience.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/login-page.md:20]:
```suggestion
The `Remember Me` checkbox allows users to opt for the application to remember the users login information. When checked, users won't need to re-enter their email and password each time they visit the login page. This is particularly convenient for returning users who want a seamless login experience.
```

File: docs/src/user/manual/login-logout/login-page.md [hunk 4]
```diff
@@ -1,34 +1,32 @@
...
 
 ### Security
 
-The `Login Page` is designed with security in mind to protect your account and sensitive information. It uses encryption protocols and best practices to ensure that your login credentials are kept secure.
+The `Login Page` is designed with security in mind to protect the users account and sensitive information. It uses encryption protocols and best practices to ensure that the users login credentials are kept secure.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/login-page.md:24]:
```suggestion
The `Login Page` is designed with security in mind to protect user accounts and sensitive information. It uses encryption protocols and best practices to ensure that users' login credentials are kept secure.
```

File: docs/src/user/manual/login-logout/login-page.md [hunk 5]
```diff
@@ -1,34 +1,32 @@
...
 
 
 ### Logging In
 
-Please ensure that you enter your email and password correctly to access your account. After clicking the `LOGIN` button, you will be redirected to the two factor authentication page within the application.
+Please ensure that the user enter the users email and password correctly to access the users account. After clicking the `LOGIN` button, the user will be redirected to the two factor authentication page within the application.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/login-page.md:29]:
```suggestion
Please ensure that users enter their email and password correctly to access their account. After clicking the `LOGIN` button, users will be redirected to the two factor authentication page within the application.
```

File: docs/src/user/manual/login-logout/login-page.md [hunk 6]
```diff
@@ -1,34 +1,32 @@
...
 
 ## Summary
-The Login Page serves as the gateway to your account, providing a secure and user-friendly way to access the application's features and functionality.
+The Login Page serves as the gateway to the users account, providing a secure and user-friendly way to access the application's features and functionality.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/login-page.md:32]:
```suggestion
The Login Page serves as the gateway to users' accounts, providing a secure and user-friendly way to access the application's features and functionality.
```

File: docs/src/user/manual/login-logout/logout.md [hunk 7]
```diff
@@ -1,19 +1,17 @@
 # Logout Page Documentation
 
-## Description
-
-The Logout Page is a secure and straightforward feature designed to allow users to log out of their account or session within your application. This page provides a clean and intuitive interface for users to initiate the logout process, ensuring that their account remains protected and their session ends securely.
+The Logout Page is a secure and straightforward feature designed to allow users to log out of their account or session within the users application. This page provides a clean and intuitive interface for users to initiate the logout process, ensuring that their account remains protected and their session ends securely.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/logout.md:3]:
```suggestion
The Logout Page is a secure and straightforward feature designed to allow users to log out of their account or session within the users' application. This page provides a clean and intuitive interface for users to initiate the logout process, ensuring that their account remains protected and their session ends securely.
```

File: docs/src/user/manual/login-logout/logout.md [hunk 8]
```diff
@@ -1,19 +1,17 @@
...
 
 ## Logout page
 
 ![Logout Form](./img/logout-1.png)
 
 
-1. **Logout Button**: To log out of the current user session, click the `LOG OUT` button. This action will terminate the user's session, ensuring they are securely logged out of the application.
+1. **Logout Button**: To log out of the current the user session, click the `LOG OUT` button. This action will terminate the users session, ensuring they are securely logged out of the application.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/logout.md:10]:
```suggestion
1. **Logout Button**: To log out of the current user session, users should click the `LOG OUT` button. This action will terminate the user's session, ensuring they are securely logged out of the application.
```

File: docs/src/user/manual/login-logout/logout.md [hunk 9]
```diff
@@ -1,19 +1,17 @@
...
 
 ## Logout Process
 
-When you click the `LOG OUT` button, the following actions occur:
+When the user click the `LOG OUT` button, the following actions occur:
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/logout.md:14]:
```suggestion
When the user clicks on the `LOG OUT` button, the following actions occur:
```

File: docs/src/user/manual/login-logout/logout.md [hunk 10]
```diff
@@ -1,19 +1,17 @@
...
 
-1. The system securely terminates your current session.
-2. You will be logged out of your account, and any temporary data related to your session will be cleared.
+1. The system securely terminates the users current session.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/logout.md:16]:
```suggestion
1. The system securely terminates the user's current session.
```

File: docs/src/user/manual/login-logout/logout.md [hunk 11]
```diff
@@ -1,19 +1,17 @@
...
+2. User will be logged out of the users account, and any temporary data related to the users session will be cleared.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/logout.md:17]:
```suggestion
2. The user will be logged out of their account, and any temporary data related to the user's session will be cleared.
```

File: docs/src/user/manual/register/register-page.md [hunk 12]
```diff
@@ -1,45 +1,43 @@
 # Registration Page Documentation
 
-## Description
-
-The Registration Page is a critical component of our platform, allowing users to create their accounts and gain access to the system. This page provides a straightforward and secure registration form for new users to sign up. It collects essential user information, including first name, last name, email address, password, and confirmation of the password.
+The Registration Page is a critical component of the platform, allowing users to create their accounts and gain access to the system. This page provides a straightforward and secure registration form for new users to sign up. It collects essential the user information, including first name, last name, email address, password, and confirmation of the password.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/register/register-page.md:3]:
```suggestion
The Registration Page is a critical component of the platform, allowing users to create their accounts and gain access to the system. This page provides a straightforward and secure registration form for new users to sign up. It collects essential user information, including first name, last name, email address, password, and confirmation of the password.
```

File: docs/src/user/manual/register/register-page.md [hunk 13]
```diff
@@ -1,45 +1,43 @@
...
 
 ## Registration Form
 
-The Registration Form is designed to gather the necessary information for creating a user account.
+The Registration Form is designed to gather the necessary information for creating the user account.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/register/register-page.md:7]:
```suggestion
The Registration Form is designed to gather the necessary information for creating a user account.
```

File: docs/src/user/manual/register/register-page.md [hunk 14]
```diff
@@ -1,45 +1,43 @@
...
 
 ![Registration Form](./img/register-page-1.png)
 
-1. **First Name**: Enter your first name in this field. This is a required field and should contain your legal first name.
-2. **Last Name**: Provide your last name in this field. Similar to the first name, this is also a required field.
-3. **Email Address**: Input your valid email address. It must be a unique and valid email that you have access to. This email will be used for communication and account recovery.
-4. **Password**: Create a strong and secure password for your account. Your password should consist of a minimum of twelve characters, including at least one lowercase letter, one uppercase letter, one numeric character, and one special character.
-5. **Confirm Password**: Re-enter the password you provided in the previous field to ensure accuracy.
-6. **Register Button**: Click this button to submit the registration form and create your account.
+1. **First Name**: Enter the users first name in this field. This is a required field and should contain the users legal first name.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/register/register-page.md:11]:
```suggestion
1. **First Name**: Enter the user's first name in this field. This is a required field and should contain the user's legal first name.
```

File: docs/src/user/manual/register/register-page.md [hunk 15]
```diff
@@ -1,45 +1,43 @@
...
+2. **Last Name**: Provide the users last name in this field. Similar to the first name, this is also a required field.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/register/register-page.md:12]:
```suggestion
2. **Last Name**: Provide the user's last name in this field. Similar to the first name, this is also a required field.
```

File: docs/src/user/manual/register/register-page.md [hunk 16]
```diff
@@ -1,45 +1,43 @@
...
+3. **Email Address**: Input the users valid email address. It must be a unique and valid email that the user have access to. This email will be used for communication and account recovery.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/register/register-page.md:13]:
```suggestion
3. **Email Address**: Input the user's valid email address. It must be a unique and valid email that the user has access to. This email address will be used for communication and account recovery.
```

File: docs/src/user/manual/register/register-page.md [hunk 17]
```diff
@@ -1,45 +1,43 @@
...
+4. **Password**: Create a strong and secure password for the users account. The users password should consist of a minimum of twelve characters, including at least one lowercase letter, one uppercase letter, one numeric character, and one special character.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/register/register-page.md:14]:
```suggestion
4. **Password**: Create a strong and secure password for the user's account. Users' passwords should consist of a minimum of twelve characters, including at least one lowercase letter, one uppercase letter, one numeric character, and one special character.
```

File: docs/src/user/manual/register/register-page.md [hunk 18]
```diff
@@ -1,45 +1,43 @@
...
+5. **Confirm Password**: Re-enter the password the user provided in the previous field to ensure accuracy.
+6. **Register Button**: Click this button to submit the registration form and create the users account.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/register/register-page.md:16]:
```suggestion
6. **Register Button**: Click this button to submit the registration form and create the user's account.
```

File: docs/src/user/manual/register/register-page.md [hunk 19]
```diff
@@ -1,45 +1,43 @@
...
 
 ### Password Guidelines
 
-To ensure the security of your account, we recommend the following password guidelines:
+To ensure the security of the users account, we recommend the following password guidelines:
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/register/register-page.md:20]:
```suggestion
To ensure the security of the user's account, we recommend the following password guidelines:
```

File: docs/src/user/manual/register/register-page.md [hunk 20]
```diff
@@ -1,45 +1,43 @@
...
 
 - Use a minimum of 12 characters.
 - Include a uppercase (A-Z).
 - Include a lowercase (a-z).
 - Use at least one number (0-9).
 - Include at least one special character (e.g., !, @, #, $, %, etc.).
 
-### Creating Your Account
+### Creating User Account
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/register/register-page.md:28]:
```suggestion
### Creating a User Account
```

File: docs/src/user/manual/register/register-page.md [hunk 21]
```diff
@@ -1,45 +1,43 @@
...
 
 1. Fill out all the required fields in the registration form.
-2. Double-check your provided information to ensure accuracy, especially your email and password.
-3. Click the "Register" button to submit the form.
+2. Double-check the users provided information to ensure accuracy, especially the users email and password.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/register/register-page.md:31]:
```suggestion
2. Double-check the user's provided information to ensure accuracy, especially the user's email and password.
```

File: docs/src/user/manual/register/register-page.md [hunk 22]
```diff
@@ -1,45 +1,43 @@
...
+3. Click on the `REGISTER` button to submit the form.
 
 ### Account Confirmation
 
-After successfully registering, you may receive an email confirmation to verify your email address. Please follow the instructions in the email to complete the registration process.
+After successfully registering, the user may receive an email confirmation to verify the users email address. Please follow the instructions in the email to complete the registration process.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/register/register-page.md:36]:
```suggestion
After successfully registering, the user should receive an email confirmation to verify their email address. Please follow the instructions in the email to complete the registration process.
```

File: docs/src/user/manual/register/register-page.md [hunk 23]
```diff
@@ -1,45 +1,43 @@
...
 
 ### Security and Privacy
 
-We take your security and privacy seriously. Your information is encrypted and stored securely, and we do not share your personal data with third parties without your consent.
+We take the users security and privacy seriously. The users information is encrypted and stored securely, and we do not share the users personal data with third parties without the users consent.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/register/register-page.md:40]:
```suggestion
We take user security and privacy seriously. User information is encrypted and stored securely, and we do not share the user's personal data with third parties without the user's consent.
```

File: docs/src/user/manual/register/register-page.md [hunk 24]
```diff
@@ -1,45 +1,43 @@
...
 
 ## Summary
-The Registration Page is your gateway to becoming a registered user, providing you with access to the platform's features and functionalities. If you encounter any issues during the registration process, please contact our support team for assistance.
+The Registration Page is the users gateway to becoming a registered user, providing the user with access to the platform's features and functionalities. If the user encounter any issues during the registration process, please contact the support team for assistance.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/register/register-page.md:43]:
```suggestion
The Registration Page is the user's gateway to becoming a registered user, providing the user with access to the platform's features and functionalities. If the user encounters any issues during the registration process, they should contact the support team for assistance.
```

File: docs/src/user/manual/user-profile/notifications.md [hunk 25]
```diff
@@ -31,7 +29,7 @@ Functionalities offered on the Notifications Page:
       * When the delete icon is clicked a popup modal is evoked prompting if the user is sure of the actions they wish to take.
 
 6. Rows per page: The user is able to choose the number of notifications they prefer to see on a page (The minimum is 5).
-7. Pagination Link: The user is able to see the current page they're own. They are also able to navigate to previous and next pages using the nav links.
+7. Pagination Link: The user is able to see the current page they are own. They are also able to navigate to previous and next pages using the nav links.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/user-profile/notifications.md:32]:
```suggestion
7. Pagination Link: The user is able to see the current page they are on. They are also able to navigate to previous and next pages using the navigation links.
```

File: docs/src/user/manual/user-profile/reminders.md [hunk 26]
```diff
@@ -33,7 +31,7 @@ Other functionalities offered on the Reminders Page:
       * When the delete icon is clicked a popup modal is evoked prompting if the user is sure of the actions they wish to take.
 
 7. **Rows per page**: The user is able to choose the number of reminders they prefer to see on a page (The minimum is 5).
-8. **Pagination Link**: The user is able to see the current page they're own. They are also able to navigate to previous and next pages using the nav links.
+8. **Pagination Link**: The user is able to see the current page they are own. They are also able to navigate to previous and next pages using the nav links.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/user-profile/reminders.md:34]:
```suggestion
8. **Pagination Link**: The user is able to see the current page they are on. They are also able to navigate to previous and next pages using the navigation links.
```

File: docs/src/user/manual/user-profile/reminders.md [hunk 27]
```diff
@@ -42,7 +40,7 @@ Other functionalities offered on the Reminders Page:
 1. **Title**: The user should provide the reminder title.
 2. **Date and Time**: The user should schedule the reminder.
 3. **Reminder**: The user should provide the description of the reminder.
-4. **Reminder Type**: Based on the user role within that specific organisation the user is able to specify if the reminder is for all organisation members or it's a personal reminder.
+4. **Reminder Type**: Based on the user role within that specific organisation the user is able to specify if the reminder is for all organisation members or it is a personal reminder.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/user-profile/reminders.md:43]:
```suggestion
4. **Reminder Type**: Based on the user role within that specific organisation the user is able to specify if the reminder is for all organisation members or if it is a personal reminder.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 8 sec

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 14 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 54 sec

**Comment by ketankartoza** [comment]
@Jeremy-Prior Please review this PR first because all other PRs rely on this PR.
Thanks!

File: docs/src/user/manual/login-logout/login-2fa-page.md [hunk 28]
```diff
@@ -1,21 +1,18 @@
...
+3. **Lost Your Device?**: Below the `Authenticate` button, the user will see a section titled `Lost your device?`.This section provides guidance for users who have lost access to their 2FA device. If the user no longer has access to their authentication app, they can follow the instructions provided here to regain access to their account. Typically, this involves contacting the user's account administrator, or support, for further assistance.
 
 ## Two-Factor Authentication Security
 
-Two-Factor Authentication (2FA) is a critical security measure that significantly reduces the risk of unauthorized access to your account. It is highly recommended to enable and use 2FA to protect your data and ensure the integrity of your account.
+Two-Factor Authentication (2FA) is a critical security measure that significantly reduces the risk of unauthorized access to the user's account. It is highly recommended to enable and use 2FA to protect a user's data and ensure the integrity of their account.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/login-2fa-page.md:15]:
```suggestion
Two-Factor Authentication (2FA) is a critical security measure that significantly reduces the risk of unauthorised access to the user's account. It is highly recommended to enable and use 2FA to protect a user's data and ensure the integrity of their account.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 45 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 32 sec

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 38 sec

**Pull request merged**